racket-cas/racket-cas.rkt

```diff
@@ -295,6 +293,8 @@
     [(bool1 bool2) (and bool1 (not bool2))] ; #t < #f
     [(bool u)      #t]
     [(u bool)      #f]
+    [((Imaginary u) v) #f]
+    [(u (Imaginary v)) #t]
```

Comment:
The predicate `<<` needs to be fast - so the representation is chosen in such a way that all comparisons can be look at the 1-2 first elements of an expressions.